.github/workflows/concrete_python_test_macos.yml

```diff
@@ -121,7 +121,6 @@ jobs:
           find .testenv/lib/python3.10/site-packages -name 'lib*omp5.dylib' | xargs -n 1  ln -f -s $(pwd)/.testenv/lib/python3.10/site-packages/concrete/.dylibs/libomp.dylib
 
           cp -R $GITHUB_WORKSPACE/frontends/concrete-python/tests ./tests
-          pytest tests -svv -n auto --key-cache "/tmp/KeySetCache" -m "not dataflow"
-          rm -rf /tmp/KeySetCache
+          pytest tests -svv -n auto --key-cache "./KeySetCache" -m "not dataflow"
```

Comment:
I remember there was some issue on macos with disk full.
Is it solved ? Also there is a proper mecanism for KeySetCache clearing based on recently touched file.

Reply:
This is prevent saving keyset cache across runs, we use a relative path in a temporary folder that is cleaned after tests

Reply:
but I just realize if the tests fails the temp folder is not cleaned. I fix that.